app/javascript/listeners/index.js

```diff
@@ -1,25 +1,34 @@
 import consumer from "../channels/consumer";
 import { callback, getChannelName } from "../channels/client_channel";
 import { initNestedAttributesListeners } from "../lib/nested_attributes";
-import helpers from "helpers";
+import helpers from "../helpers";
 import { initTakeActionOnChangeHandlers } from "../lib/dynamic_take_action_changes";
 import { initMetricsTableSortAndFilter } from "../lib/metrics_table_sort";
+import { documentSubmittingIndicator } from "../lib/document_submitting_indicator";
 
-export function initListeners() {
-    document.addEventListener("DOMContentLoaded", function() {
-        if (window.appData.controller_action == "Hub::Users::InvitationsController#edit") {
-            helpers.setDefaultTimezone();
-        }
-        if (window.appData.controller_action == "Hub::MessagesController#index") {
-            consumer.subscriptions.create(getChannelName(window.location.href), callback);
-        }
+const Listeners =  (function(){
+    return {
+        init: function () {
```

Comment:
I refactored this to use the IFFE pattern with an init closure that we use a lot in other places. No real reason other than to follow a pattern that there seems to be consensus on and give some symmetry to our implementations.

Reply:
Makes sense, but as I wrote earlier, I don't actually love the pattern. :) Not a blocker; symmetry is fine.